tasks/db_deploy/install/orca_sql.py

```diff
@@ -818,32 +804,26 @@ def reconcile_s3_object_partition_sql(partition_name: str) -> TextClause:
     Returns:
         (sqlalchemy.sql.element.TextClause): SQL for creating reconcile_s3_object partition table.
     """
-    try:
-        if not re.match("^[\w+]+$", partition_name):  # noqa: W605
-            raise ValueError(f"Table name {partition_name} is invalid.")
-    except TypeError:
-        raise ValueError("Table name must be a string and cannot be None.")
```

Comment:
why are we removing this check?

Reply:
It's being moved to the calling function to remove business logic from the sql functions.